### PR TITLE
ref(issueDetails): Use new Tabs component

### PR DESCRIPTION
**Pull request by vuluongj20**
**Before:**
Screen Shot 2022-09-21 at 5 59 15 PM

**After:**
Screen Shot 2022-09-21 at 6 00 28 PM

### CODE REVIEW

File: static/app/routes.tsx [hunk 1]
```diff
@@ -1313,15 +1313,15 @@ function buildRoutes() {
         }}
       />
       <Route
-        path="tags/"
+        path={TabPaths[Tab.TAGS]}
         component={make(() => import('sentry/views/organizationGroupDetails/groupTags'))}
         props={{
           currentTab: Tab.TAGS,
```
Comment (evanpurkhiser) [inline static/app/routes.tsx:1319]: Possible to delete these props now?

Comment (vuluongj20) [reply static/app/routes.tsx:1319]: Working on that now

Comment (vuluongj20) [reply static/app/routes.tsx:1319]: But I think so

Comment (vuluongj20) [reply static/app/routes.tsx]: Done! https://github.com/getsentry/sentry/pull/39137/commits/62ef51613bc27579d5373bbeb770540fbdae569b

File: static/app/routes.tsx [hunk 2]
```diff
@@ -1276,214 +1276,126 @@ function buildRoutes() {
...
       />
       <Route
-        path="replays/"
+        path={TabPaths[Tab.REPLAYS]}
         component={make(
           () => import('sentry/views/organizationGroupDetails/groupReplays')
         )}
-        props={{
-          currentTab: Tab.REPLAYS,
-          isEventRoute: false,
```
Comment (evanpurkhiser) [inline static/app/routes.tsx:1291]: Actually do we still need this `isEventRoute` thing for something?

File: static/app/views/organizationGroupDetails/groupDetails.tsx [hunk 3]
```diff
@@ -193,21 +195,18 @@ class GroupDetails extends Component<Props, State> {
...
-    const {routes, organization} = this.props;
+    const {organization, router, routes} = this.props;
     const {event} = this.state;
 
-    // All the routes under /organizations/:orgId/issues/:groupId have a defined props
-    const {currentTab, isEventRoute} = routes[routes.length - 1].props as {
-      currentTab: Tab;
-      isEventRoute: boolean;
-    };
+    const currentRoute = routes[routes.length - 1];
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/groupDetails.tsx:201]: CLEAN!

File: static/app/views/organizationGroupDetails/groupDetails.tsx [hunk 4]
```diff
@@ -193,21 +195,18 @@ class GroupDetails extends Component<Props, State> {
   }
 
   getCurrentRouteInfo(group: Group): {baseUrl: string; currentTab: Tab} {
-    const {routes, organization} = this.props;
+    const {organization, router, routes} = this.props;
     const {event} = this.state;
 
-    // All the routes under /organizations/:orgId/issues/:groupId have a defined props
-    const {currentTab, isEventRoute} = routes[routes.length - 1].props as {
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/groupDetails.tsx:200]: This is a million times better than this, this is great

File: static/app/routes.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/routes.tsx:1291]: It's only used to generate `baseUrl` in `groupDetails.tsx` - https://github.com/getsentry/sentry/blob/master/static/app/views/organizationGroupDetails/groupDetails.tsx#L198-L207. I found a different way to do that without relying on this prop.

Comment (evanpurkhiser) [reply static/app/routes.tsx:1291]: yessssssssss

**Review by evanpurkhiser — APPROVED** [review]
Love this

**Pull request merged**